pkg/messages/reason_code.go

```diff
@@ -176,4 +176,7 @@ const (
 
 	// SubnetFindFailed ...
 	SubnetFindFailed = "SubnetFindFailed"
+
+	// RC5XX
+	RC5XX = "RC:5"
```

Comment:
hope this will not cause any issue like existing string contains "RC: 500" or " RC : 501 " etc

Reply:
We have 2 ways to get RC one from the RIAAS backend we know it is RC:5xx or RC:4xx and other way is via our defined messages which does not have space. So we should be good.

Reply:
Done